flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -240,19 +242,18 @@
  * <p>FLINK modifications are at lines
  *
  * <ol>
- *   <li>Added in FLINK-29081, FLINK-28682, FLINK-33395: Lines 670 ~ 687
- *   <li>Added in Flink-24024: Lines 1463 ~ 1469
- *   <li>Added in Flink-24024: Lines 1483 ~ 1522
- *   <li>Added in Flink-37269: Lines 2239 ~ 2261
- *   <li>Added in FLINK-28682: Lines 2372 ~ 2389
- *   <li>Added in FLINK-28682: Lines 2426 ~ 2454
- *   <li>Added in FLINK-32474: Lines 2507 ~ 2509
- *   <li>Added in FLINK-32474: Lines 2513 ~ 2515
- *   <li>Added in FLINK-32474: Lines 2526 ~ 2528
- *   <li>Added in FLINK-32474: Lines 2934 ~ 2945
- *   <li>Added in FLINK-32474: Lines 3046 ~ 3080
- *   <li>Added in FLINK-34312: Lines 5827 ~ 5838
- *   <li>Added in FLINK-34057, FLINK-34058, FLINK-34312: Lines 6285 ~ 6303
+ *   <li>Added in FLINK-29081, FLINK-28682, FLINK-33395: Lines 673 ~ 690
```

Comment:
Thank you for updating all these lines. Very helpful, but I'm sure also a lot of work!